_versions/2.2/guides/kafka-schema-registry-avro.adoc

```diff
@@ -613,7 +613,7 @@ public class KafkaAndSchemaRegistryTestResource implements QuarkusTestResourceLi
         registry.start();
         Map<String, String> properties = new HashMap<>();
         properties.put("mp.messaging.connector.smallrye-kafka.apicurio.registry.url",
-                "http://" + registry.getContainerIpAddress() + ":" + registry.getMappedPort(8080) + "/apis/registry/v2");
+                "http://" + registry.getHost() + ":" + registry.getMappedPort(8080) + "/apis/registry/v2");
```

Comment:
Just to be sure: have you checked if this method is already there in the version of Testcontainers used in 2.2?

Reply:
quarkus 2.2 uses testcontainers 1.15.3 as you can see [here](https://github.com/quarkusio/quarkus/blob/2.2.0.Final/bom/application/pom.xml#L207) and `getHost` is available is that version https://github.com/testcontainers/testcontainers-java/blob/1.15.3/core/src/main/java/org/testcontainers/containers/ContainerState.java#L58-L61